src/Components/Lifecycle/lifecycle.tsx

```diff
@@ -60,39 +60,14 @@ const lifecycleChartData = [
 
 type dataAppLifeCycleChanges = {
   data: AppLifecycleChanges[];
-
-}
-
-type AppLifecycleChanges = {
```

Comment:
You did absolutely nothing wrong here, but I moved these out of the file since we will need these in other files later. We want to have the types in one place so we can update them once whenever they change.